internal/mux/mux.go

```diff
@@ -129,6 +129,10 @@ func (m *Mux) readLoop() {
 		}
 
 		if err = m.dispatch(buf[:n]); err != nil {
+			if errors.Is(err, io.ErrClosedPipe) {
```

Comment:
I don't see a reason to have this spuriously log when we are the ones closing the packetio.Buffer.